Lib/csv.py

```diff
@@ -434,6 +443,12 @@ def has_header(self, sample):
         # finally, compare results against first row and "vote"
         # on whether it's a header
         hasHeader = 0
+
+        # special case when all columns are strings and columnTypes has been emptied
+        if not columnTypes and col_are_strings and columns > 0:
+            # If there are only columns of strings and no column types specified, we update the dictionary to store the average length of all strings
```

Comment:
```suggestion
            # If there are only columns of strings and no column types specified,
            # update the dictionary to store the average length of all strings
```

Keep to a more reasonable line length and use imperative